scripts/lib.sh

```diff
@@ -104,8 +104,8 @@ create-db() {
 
 apply-migrations() {
     create-db
-    echo "--> Applying migrations"
-    sentry upgrade --noinput
+    echo "--> Applying migrations and creating kafka topics"
```

Comment:
```suggestion
    echo "--> Applying migrations and ensuring kafka topics"
```

since they may already be created